.circleci/config.yml

```diff
@@ -75,12 +84,24 @@ workflows:
     version: 2
     build-test-deploy:
       jobs:
+        - gradle/test:
```

Comment:
I guess I was thinking that we would just do the 'test' step in the job called 'build_and_test'. Does this orb do something different?